tutor/src/screens/scores-report/late-work.jsx

```diff
@@ -177,12 +177,10 @@ export class LateWorkPopover extends React.PureComponent {
     const { content } = this;
 
     const status = this.props.task.type === 'homework' ? content.score() : content.progress();
-    const arrowOffsetTop = TH.hasAdditionalLateWork(this.props.task) ? 128 : 95;
     const popoverProps = omit(this.props, 'hide', 'task', 'show', 'columnIndex');
     return (
       <Popover
         {...popoverProps}
-        arrowOffsetTop={arrowOffsetTop}
         title={content.get('title')}
```

Comment:
nice catch, I have no idea why were were setting that

Reply:
i also deleted some styles that were making the popover taller for no reason i could figure out

Reply:
yes, I saw that, I also have no idea why we had those.  Maybe we had additional absolutely positioned content in the popover at one time?